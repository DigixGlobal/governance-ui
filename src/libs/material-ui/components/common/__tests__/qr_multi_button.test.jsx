import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import QrMultiButton from '../qr_multi_button';

describe('<QrMultiButton />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <QrMultiButton
        onScan={jest.fn}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
