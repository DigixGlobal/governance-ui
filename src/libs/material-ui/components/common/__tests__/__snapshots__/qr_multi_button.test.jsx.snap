// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QrMultiButton /> renders correctly 1`] = `
<EZModal
  content={
    <div>
      <Message
        content="Scanned 0 items"
        header="Scan QR Codes"
        icon="qrcode"
        negative={false}
        positive={false}
        warning={false}
      />
      <QrReader
        onError={[Function]}
        onScan={[Function]}
      />
    </div>
  }
  size="small"
  trigger={
    <Button
      as="button"
      basic={true}
      content="Scan QRs"
      icon="qrcode"
      onClick={[Function]}
    />
  }
/>
`;
